### PR TITLE
Operator 3.1.3

### PR DESCRIPTION
**Pull request by rjeberhard**
Resolves a pair of issues related to running well in very large Kubernetes clusters.

Note: I will create a pre-release build now and provide it to the customer for final validation before we release this version tomorrow.

### CODE REVIEW

**Review by ankedia — APPROVED** [review]

File: docs-source/content/release-notes.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ draft: false
 
 | Date | Version | Introduces backward incompatibilities? | Change |
 | --- | --- | --- | --- |
+| February 12, 2021 | v3.1.3 | no | Resolved a pair of issues related to running well in very large Kubernetes clusters. |
```
Comment (rosemarymarano) [inline docs-source/content/release-notes.md:11]: to running well -> (just a suggestion) to the operator running well

File: docs-source/content/release-notes.md [hunk 2]
```diff
@@ -34,6 +35,11 @@ draft: false
 
 ### Change log
 
+#### Operator 3.1.3
+
+* Resolved an issue that caused some WebLogic servers to fail to start in large Kubernetes clusters where Kubernetes watch notifications were not reliably delivered ([#2188](https://github.com/oracle/weblogic-kubernetes-operator/pull/2188)).
```
Comment (rosemarymarano) [inline docs-source/content/release-notes.md:40]: WebLogic servers -> WebLogic Servers

File: docs-source/content/release-notes.md [hunk 3]
```diff
@@ -34,6 +35,11 @@ draft: false
...
+* Resolved an issue that caused the operator to ignore some namespaces it was configured to managed in Kubernetes clusters that had more than 500 namespaces ([#2189](https://github.com/oracle/weblogic-kubernetes-operator/pull/2189)).
```
Comment (rosemarymarano) [inline docs-source/content/release-notes.md:41]: it was configured to managed -> it was configured to manage

**Review by rosemarymarano — COMMENTED** [review]
minor stuff

**Review by rosemarymarano — APPROVED** [review]

**Pull request merged**